.github/workflows/plugins.yml

```
@@ -1,46 +1,55 @@
 name: plugins
 on:
+# Run CI against all pushes (direct commits) and Pull Requests
   push:
-    paths:
-      - 'plugins/**'
-      - 'tests/**'
-      - 'scripts/**'
-      - '.github/workflows/plugins.yml'
-      - 'docker-compose.yml'
+#    paths:
+#      - 'plugins/**'
+#      - 'tests/**'
+#      - 'scripts/**'
+#      - '.github/workflows/plugins.yml'
+#      - 'docker-compose.yml'
   pull_request:
-    paths:
-      - 'plugins/**'
-      - 'tests/**'
-      - 'scripts/**'
-      - '.github/workflows/plugins.yml'
-      - 'docker-compose.yml'
+#    paths:
+#      - 'plugins/**'
+#      - 'tests/**'
+#      - 'scripts/**'
+#      - '.github/workflows/plugins.yml'
+#      - 'docker-compose.yml'
 
 jobs:
   sanity:
-    name: Sanity (${{ matrix.ansible }})
+    name: Sanity (Ⓐ${{ matrix.ansible }}+py${{ matrix.python }})
     strategy:
       matrix:
         ansible:
-        - stable-2.10
-        - devel
+          # - stable-2.9 # Only if your collection supports Ansible 2.9
+          - stable-2.10
+          - devel
+        python:
+          - 2.7
+          - 3.7
+          - 3.8
+          exclude:
+          - python: 3.8  # blocked by https://github.com/ansible/ansible/pull/70155
```

Looks like I suggested a poorly indented block before. This should fix it:
```suggestion
        exclude:
        - python: 3.8  # blocked by https://github.com/ansible/ansible/pull/70155
```

---

@gundalow this causes YAML parsing error ^